SecretAgent/SecretAgent/Screen/Board/View/CustomCollectionViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CustomCollectionViewCell.swift
+//  SecretAgent
+//
+//  Created by Minkyeong Ko on 2022/11/23.
+//
+
+import UIKit
+
+class CustomCollectionViewCell: UICollectionViewCell {
+    static let identifier = "CustomCollectionViewCell"
+
+    let badgeImageView = UIImageView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubview(badgeImageView)
+        badgeImageView.snp.makeConstraints { make in
+            make.trailing.equalToSuperview()
```

Comment:
p4.

```suggestion
            make.top.trailing.equalToSuperview()
```

<img width="1508" alt="image" src="https://user-images.githubusercontent.com/59143479/203930890-6093de2d-40ac-488b-b8b7-fcce55171ee2.png">

`View Hierarchy` 를 디버그 해보니까 왼쪽 사이드바에서 보라색 워닝이 뜨면서 오른쪽 사이드바에서 확인해보니 `Vertical Position is ambiguous` 라고 워닝이 떴습니다! 따라서 top을 추가해줬더니 워닝도 사라지고 뷰도 차이가 없는 것 같아서 top도 constraint를 주면 좋을 것 같습니다!!

Reply:
와 꼼꼼한 디버깅.. 바로 수정 반영하도록 하겠습니다! 
앞으로는 PR 올리기 전에 View Hierarchy 를 확인해 봐야겠네요😂 넘 감사합니다 꼼꼼 코지